doc/requirements.txt

```diff
@@ -27,7 +27,7 @@ tables
 descartes
 
 rioxarray
-git+https://github.com/davide-f/google-drive-downloader@master  # if not included will create error in docs `make html`
+googledrivedownloader>=1.1
```

Comment:
Do I get properly that the upstream version can be used for `googledrivedownloader ` as the virus scan stopped to be a problem?

Reply:
version 1.1 fixed the virus scan stuff, indeed. However, the interface between the previous version and the new one has changed. Indeed few fixes have been introduced in the workflow accordingly

Reply:
Thank you so much for that 😄